test/services/cert/certs_test.rb

```diff
@@ -17,7 +17,10 @@ def teardown
     end
 
     def test_verify_ueber_cert_no_change
-      Setting[:ssl_ca_file] = File.join("#{Katello::Engine.root}", "/test/services/cert/helpers/ca.crt")
```

Comment:
don't forget to delete the crt file.

Reply:
hmm I am using it here https://github.com/Katello/katello/pull/7627/files#diff-816d256f98117950a20b7ae031224817